core/src/main/java/fr/sncf/osrd/api/stdcm/STDCMRequest.java

```diff
@@ -81,13 +81,12 @@ public final class STDCMRequest {
     public double maximumDepartureDelay = 3600 * 2;
 
     /**
-     * How much longer can the train take to reach its destination,
-     * compared to the time it would take with an empty timetable.
-     * e.g. with a value of 2 and a fastest run time of 1h, we ignore results after 2h.
-     * Defaults to 2.
+     * Specifies the running time that the train should not exceed to reach its destination,
+     * e.g. with a value of 18000, we ignore results after 5h.
+     * Defaults to 12hrs.
      */
-    @Json(name = "maximum_relative_run_time")
-    public double maximumRelativeRunTime = 2;
+    @Json(name = "maximum_run_time")
+    public double maximumRunTime;
```

Comment:
The `/core/openapi.yaml` file needs to be updated.

It appears that this parameter isn't documented at all in the openapi, but it should be and it's a good opportunity to add it